README

```diff
@@ -12,6 +12,7 @@ ARM disk image for use with various hardware targets.
  - Turn SE Linux on and off
  - Remove the root password
  - Update U-Boot for a Target
+ - Setup wifi connection
```

Comment:
Can we add a little more here around wpa-psk and sae options. Most won't know that sae is WPA3 for example.

Reply:
added details for easier undertsanding